code/ship/shipfx.cpp

```diff
@@ -2176,6 +2176,20 @@ void shipfx_do_lightning_arcs_frame( ship *shipp )
 
 						shipp->arc_type[j] = MARC_TYPE_SHIP;
 
+						if (arc_info->width > 0.0) {
+							shipp->arc_width[j] = arc_info->width;
+						} else {
+							// same as MARC_TYPE_DAMAGED
```

Comment:
What does `// same as MARC_TYPE_DAMAGED` actually mean here?  The comment deleted from modelrender.cpp, 
`// try and scale the size a bit so that it looks equally well on smaller vessels`, was reasonably descriptive, but I'm not sure how to interpret the new comment.

Reply:
It uses the same width calculation `MARC_TYPE_DAMAGED` arcs use in `model_render_add_lightning`, I'll comment to that effect.